content/en/blog/2023/jmx-metric-insight/index.md

```diff
@@ -111,10 +111,9 @@ that our Kafka installation is working as expected.
 ### Export metrics to Prometheus
 
 The metrics can be exported by any of the supported metric exporters, to a
-backend of your choice. The full list of exporters and their configuration
-options can be found
-[here](https://github.com/open-telemetry/opentelemetry-java/blob/main/sdk-extensions/autoconfigure/README.md#exporters).
-For instance, you can export the metrics to an OTel collector using the OTLP
+backend of your choice. For the full list of exporters and their configuration
+options, see [Configure the SDK](/docs/languages/java/configuration/). For
+instance, you can export the metrics to an OTel collector using the OTLP
```

Comment:
@jack-berg @open-telemetry/java-approvers is this change ok?

Reply:
Better to link to here: https://opentelemetry.io/docs/languages/java/configuration/#properties-exporters